setup.py

```diff
@@ -33,6 +37,11 @@ def reqs(file_path):
     entry_points={"console_scripts": ["gretel=gretel_client.cli.cli:cli"]},
     install_requires=reqs("requirements.txt"),
     tests_require=reqs("test-requirements.txt"),
+    extras_require={
+        "s3": aws_deps,
+        "gcs": gcs_deps,
+        "azure": azure_deps,
```

Comment:
These should be cloud specific instead of storage specific
```suggestion
        "aws": aws_deps,
        "gcp": gcs_deps,
        "azure": azure_deps,
```